### PR TITLE
docs: add password reset flow

### PR DESCRIPTION
**Pull request by tas17**

### CODE REVIEW

File: doc/source/changelog.rst [hunk 1]
```diff
@@ -5,6 +5,7 @@ Changelog
 ------
 
 -  Handle category dtype
+-  Fix download of reports
```
Comment (tcrasset) [inline doc/source/changelog.rst:8]: Is this from a previous commit that we forgot ?

File: doc/source/tutorial.rst [hunk 2]
```diff
@@ -296,3 +300,31 @@ to make it safe.
...
    print(avatars_df.head())
+
+Reset your password
+-------------------
+
+**NB**: This section is only available if the use of emails to login is
+activated in the global configuration. It is not the case by default.
+
+If you forgot your password or it you need to set one, first notify that
+you forgot it:
```
Comment (tcrasset) [inline doc/source/tutorial.rst:311]:
```suggestion
If you forgot your password or if you need to set one, first notify that
you forgot it:
```

Comment (tcrasset) [inline doc/source/tutorial.rst:311]: What do you mean by "notify" ?

File: docs/tutorial.md [hunk 3]
```diff
@@ -257,3 +257,27 @@ sensitive_unshuffled_avatars_df = pd.read_csv(
...
+
+**NB**: This section is only available if the use of emails to login is activated in the global configuration. It is not the case by default.
+
+If you forgot your password or it you need to set one, first notify that you forgot it:
+
+```python
+from avatars.client import apiclient
+
+client = apiclient(base_url=os.environ.get("BASE_URL"))
+client.forgotten_password("yourmail@mail.com")
```
Comment (tcrasset) [inline docs/tutorial.md:271]:
```suggestion
from avatars.client import ApiClient

client = ApiClient(base_url=os.environ.get("BASE_URL"))
client.forgotten_password("yourmail@mail.com")
```

Comment (tcrasset) [reply docs/tutorial.md:271]: Also, don't they need to use `ForgottenPasswordRequest` ? 
Same with `ResetPasswordRequest` ?

**Review by tcrasset — CHANGES_REQUESTED** [review]
The tutorial should be runnable as is by `make test-integration` in `avatar-python`. I fear that, if the email authentication was not setup, these endpoints will make it fail.

File: doc/source/changelog.rst (hunk 1 above)
Comment (tas17) [reply doc/source/changelog.rst:8]: Indeed. Maybe we should remove it ?

File: docs/tutorial.md (hunk 3 above)
Comment (tas17) [reply docs/tutorial.md:271]: Nope, I did not update the code yet but it's not necessary :)

**Pull request merged**